test/test_tamper_evident.py

```diff
@@ -73,7 +73,7 @@ def test_editing_is_detected(self):
         self.assertFalse(tef.validate())
 
     def test_oneline_file_is_detected(self):
-        filename = self.write_tamper_evident(b"Am I OK?")
+        filename = self.write_tamper_evident("Am I OK?")
```

Comment:
Gah... I'm not sure why I changed it to explicit unicode. Let me try and jog my memory, and I'll get back to you.